.github/workflows/screener-run.yml

```diff
@@ -42,14 +42,14 @@ jobs:
         uses: dawidd6/action-download-artifact@v2
         with:
           workflow: screener-build.yml
-          workflow_conclusion: success
```

Comment:
**Q**: why `workflow_conclusion` is removed?
**A**: _Do not specify pr, commit, branch, run_id together or workflow_conclusion and run_id together. Pick just one of each or none._ https://github.com/dawidd6/action-download-artifact#usage